README.md

```diff
@@ -1,2 +1,150 @@
+
+However, the vast majority of interactions between people, organizations, and things have simpler requirements. When Alice(Corp|Device) and Bob want to interact, there are exactly and only 2 parties in the world who should care: Alice and Bob. Instead of arbitrary parties needing to resolve their DIDs, only Alice and Bob do.
+
+One to one interactions are excellent fit for [did:key] identifiers, however they suffer from key discovery problem and introduce additional challenges when interaction sessions span across more than two devices.
+
+Mailto DIDs are designed to be used in conjunction with [did:key] and facilitate bootstrapping sessions between two parties that span across multiple devices.
+
+Mailto DIDs are more accessible alternative to [did:web] and [did:dns] because a lot more people have an email address than there are people with [did:web] or [did:dns] identifier or skills to acquire one.
+
+Mailto [did document]s are also verifiable offline, when [domain key][] of the email address is known, implying significantly less network requests than with most other DIDs.
+
+## The `did:mailto` Format
+
+The format for the `did:key` method conforms to the [DID-CORE] specification and is encoding of the [email address]. It consists of the `did:mailto:` prefix, followed by the `domain` part of an email address, `:` character and percent encoded `local-part` of the email address.
+
+The ABNF definition can be found below. The formal rules describing valid `domain-name` syntax are described in [RFC1035], [RFC1123], [RFC2181]. The `domain-name` and `user-name` corresponds to `domain` and `local-part` respectively of the email address described in [RFC2822]. All "mailto" DIDs MUST conform to the DID Syntax ABNF Rules.
+
+```abnf
+did       = "did:mailto:" domain-name ":" user-name
+user-name = 1*idchar
+idchar    = ALPHA / DIGIT / "." / "-" / "_" / pct_enc
+pct_enc   = "%" HEXDIG HEXDIG
+```
+
+##### EXAMPLE 1. A jsmith@example.com
+
+```
+did:mailto:example.com:jsmith
+```
+
+##### EXAMPLE 2 tag+alice@web.mail
+
+```
+did:mailto:web.mail:tag%2Balice
+```
+
+## Operations
+
+The following section outlines the DID operations for the `did:mailto` method.
+
+### Create (Register)
+
+There is intentionally no specification for publishing `did:mailto` documents as single source of truth are not implied. Same `did:mailto` identifier MAY (intentionally) correspond to different [DID document] in different sessions.
+
+Creating a `did:mailto` document is done by:
+
+1. Creating a [did:key] identifier.
+2. Sending an email from [DID Subject] email address with a [key authentication] message in the subject.
+
+#### Key Authentication
+
+Key authentication message must conform to the following ABNF definition, which refers to `did-key-format` defined in [did:key] specification
```

Comment:
It may be good to make a distinction between 'key authentication' and 'subject aliasing'. DID documents support both, but they're not identical. It looks like this section here, because it mentions 'I am also known as' seems like the human-readable instruction is more about subject aliasing and did-core [`alsoKnownAs`](https://www.w3.org/TR/did-core/#also-known-as).

"Key Authentication" could be confused with did-core's `Authentication` verification relationship. This is how a did can use its did document to enumerate the ways (e.g. a signing key) someone can authenticate as the did.

If you alias from your `did:mailto:ab:c` via alsoKnownAs to `did:key:abc`, you're authorizing `did:key` for all verification relationships, not just [`authentication` but also `assertionMethod`, `capabilityInvocation`, and `capabilityDelegation`](https://w3c-ccg.github.io/did-method-key/#document-creation-algorithm)